### ISSUE TITLE
Queue does not work

### ISSUE THREAD

**Issue by Asetss** [SimpleSoftwareIO/simple-sms#14]
code 

``` php
SMS::queue('simple-sms::welcome', $data, function($sms) {
    $sms->to('+15555555555');
});
```

I receive an error
Class 'Illuminate\Support\SerializableClosure' not found

what went wrong?

**Comment by SimplyCorey** [comment]
Which Laravel are you using?

**Comment by Asetss** [comment]
@SimplyCorey Yes. I use the latest version. 

no such file on 'Illuminate\Support\SerializableClosure'
https://github.com/illuminate/support

**Comment by SimplyCorey** [comment]
I would try running `composer dump-autoload -o`.  It seems like your autoload file is not build correctly.

**Comment by Asetss** [comment]
Tried. 
1.I removed from my dump and 
2.Deleted vendor 
3.Command composer update. 

Does not work :|

**Comment by SimplyCorey** [comment]
I'll take a look at this.  Can you include the code you were using please?

**Comment by Asetss** [comment]
Ok. What code?

**Comment by SimplyCorey** [comment]
I've found the issue.  I will push a bug fix tomorrow when I finish up some other items for this project as well.

**Comment by Asetss** [comment]
Excellent. Will wait.

**Comment by SimplyCorey** [comment]
Life's been a little busy these past few days.  I am testing the fix and will release 2.1 shortly!  Sorry!

**Comment by Asetss** [comment]
Thank you.

**Comment by palokankare** [comment]
@SimplyCorey Do you have plans to release 2.1?

**Comment by SimplyCorey** [comment]
@Asetss @kallepa  I will be releasing this tomorrow!  I have been enjoying the holiday with my family; sorry for the delays guys!

**Comment by SimplyCorey** [comment]
Fixed in 2.1.0
